<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 209, 122);
background-color: rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>         Chào mừng bạn đã đến với Nha Khoa IoT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../.designer/backup/logo-illustration-of-a-combination-of-teeth-and-toothbrush-very-suitable-for-dental-companies-free-vector.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>911</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 132, 228);
background-color: rgb(192, 191, 188);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Lịch</string>
     </attribute>
     <widget class="QLabel" name="l01">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(98, 160, 234);
font: 20pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Lịch hẹn sắp tới của bạn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>561</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(26, 95, 180);</string>
      </property>
     </widget>
     <widget class="QListWidget" name="showLich">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>200</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Push">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>320</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color: rgb(53, 132, 228);</string>
      </property>
      <property name="text">
       <string>Lưu thay đổi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateinf">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 132, 228);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Đăng kí mới lịch hẹn mới</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="gioHen">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>118</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="DoctorName">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>60</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nguyễn Minh Quân</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Đoàn Quang Lưu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lê Quang Huy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>do</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Chọn bác sỹ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Chọn thời gian</string>
      </property>
     </widget>
     <zorder>gioHen</zorder>
     <zorder>l01</zorder>
     <zorder>calendarWidget</zorder>
     <zorder>showLich</zorder>
     <zorder>Push</zorder>
     <zorder>updateinf</zorder>
     <zorder>DoctorName</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lịch sử</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>701</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 132, 228);</string>
      </property>
      <column>
       <property name="text">
        <string>Nha sỹ </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dịch vụ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thời gian</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Số lượng</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thành tiền</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Dịch vụ</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>701</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 132, 228);</string>
      </property>
      <column>
       <property name="text">
        <string>Dịch vụ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mô tả</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Giá</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tài khoản</string>
     </attribute>
     <widget class="QPushButton" name="signOut">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>390</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Đăng xuất</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Cập nhật tài khoản</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ngày </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Số điện thoại</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mật khẩu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>240</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Năm</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nữ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lê Hoàng Anh</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Tháng</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Tên tài khoản</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 132, 228);
font: 75 13pt &quot;Ubuntu Condensed&quot;;
font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>Trang chủ</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu2">
    <property name="title">
     <string>Lịch sử khám</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu3">
    <property name="title">
     <string>Dịch vụ</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu4">
    <property name="title">
     <string>Tài khoản</string>
    </property>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu2"/>
   <addaction name="menu3"/>
   <addaction name="menu4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
